### PR TITLE
Update README for alpha release

### LINKED ISSUE #63: Update README to better reflect the problem Fondant aims to solve and the solution it provides

### PR DESCRIPTION
**Pull request by RobbeSneyders**
Fixes #63 

It still contains some TODOs, mostly for parts that are not ready yet, but this already shows the outline I'm aiming for. Feedback welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,146 +1,239 @@
...
+**Fondant is the platform where we meet to build and share data preparation workflows.**
 
-```
-pip install git+https://github.com/ml6team/fondant.git
-```
+Fondant offers a framework to build **composable data preparation pipelines, with reusable 
+components, optimized to handle massive datasets**. Stop building from scratch, and start 
+leveraging reusable components instead to:
+- Extend your data with public datasets
+- Generate new modalities using captioning, segmentation, translation, image generation, ...
```
Comment (NielsRogge) [inline README.md:39]: this is a bit of a weird sentence, you mean using captioning models?

File: README.md [hunk 2]
```diff
@@ -1,146 +1,239 @@
...
+- Distill knowledge from existing foundation models
+- Filter out low quality data
```
Comment (NielsRogge) [inline README.md:41]:
```suggestion
- Filter out low quality data
- Deduplicate data
```

File: README.md [hunk 3]
```diff
@@ -1,146 +1,272 @@
...
+- :mag_right: CLIP  
+- :scissors: Segment Anything (SAM)  
+- :heavy_plus_sign: And many more
 
-Fondant focuses on data preparation to fine-tune these models.
+## :mouse_trap: Why Fondant?
 
-## Installation
+Foundation models simplify inference by solving multiple tasks across modalities with a simple
+prompt-based interface. But what they've gained in the front, they've lost in the back.
```
Comment (Fematich) [inline README.md:29]: Clear!

File: README.md [hunk 4]
```diff
@@ -1,146 +1,272 @@
...
+We believe that **innovation is a group effort**, requiring collaboration. While the community has 
+been building and sharing models, everyone is still building their data preparation from scratch.
+**Fondant is the platform where we meet to build and share data preparation workflows.**
 
-```
-pip install git+https://github.com/ml6team/fondant.git
-```
+Fondant offers a framework to build **composable data preparation pipelines, with reusable 
+components, optimized to handle massive datasets**. Stop building from scratch, and start 
+leveraging reusable components instead to:
```
Comment (Fematich) [inline README.md:39]: Proposal to shorten sentence to:
"Stop building from scratch, and start reusing components to:"

File: README.md [hunk 5]
```diff
@@ -1,146 +1,272 @@
...
+| ![input image](docs/art/interior_design_controlnet_input1.png) | ![output image](docs/art/interior_design_controlnet_output1.jpg) |
+
+Want to try out the resulting model yourself, head over to our 
+[Hugging Face space](https://huggingface.co/spaces/ml6team/controlnet-interior-design)!
+
+### Fine-tuning Stable Diffusion
+
+Using our 
+[example pipeline to fine-tune Stable Diffusion](examples/pipelines/finetune_stable_diffusion) 
+allows you create models that generate better images within a specific domain. All you need to
```
Comment (Fematich) [inline README.md:76]: typo, missing "to": allows you "to" create models that generate better images within a specific domain.

File: README.md [hunk 6]
```diff
@@ -1,146 +1,272 @@
...
+### Fine-tuning ControlNet
+
+Our 
+[example pipeline to generate data for ControlNet fine-tuning](examples/pipelines/controlnet-interior-design) 
+allows you to create models that you can control using inpainting, segmentation, and 
+regeneration. All you need to get started is a set of prompts describing the type of images to 
+generate.
+
+Using our ControlNet model fine-tuned on interior design images for instance allows you to generate 
+the room of your dreams:
```
Comment (Fematich) [inline README.md:63]: This sentence was a bit harder to read, 2 alternatives:
- "For instance, using our ControlNet model fine-tuned on interior design images, allows you to generate the room of your dreams:"
- "For instance, fine-tuning ControlNet on interior design images allows you to generate the room of your dreams:"

**Review by Fematich — APPROVED** [review]
Added some suggestions, otherwise very clear 👍 !

**Pull request merged**